### PR TITLE
Add branch protection

### PR DESCRIPTION
**Pull request by wilyle**
Add branch protection rules to Ibeji repos. These are based on the branch protection rules defined for Chariott

### CODE REVIEW

File: otterdog/eclipse-ibeji.jsonnet [hunk 1]
```diff
@@ -21,8 +21,15 @@ orgs.newOrg('eclipse-ibeji') {
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
+      description: "freyja project",
       secret_scanning_push_protection: "disabled",
```
Comment (devkelley) [inline otterdog/eclipse-ibeji.jsonnet]: Either this PR or a separate one, but 'secret_scanning_push_protection' should either be removed and/or enabled.

**Comment by netomi** [comment]
the PR LGTM, just some food for thought. Some projects defined a template for their branch protection rules that they would like to use for all repos so that the settings dont have to be repeated. Take a look here:

https://github.com/eclipse-vertx/.eclipsefdn/blob/main/otterdog/eclipse-vertx.jsonnet#L3

this might be of interest for you at some time.

**Comment by wilyle** [comment]
> the PR LGTM, just some food for thought. Some projects defined a template for their branch protection rules that they would like to use for all repos so that the settings dont have to be repeated. Take a look here:
> 
> https://github.com/eclipse-vertx/.eclipsefdn/blob/main/otterdog/eclipse-vertx.jsonnet#L3
> 
> this might be of interest for you at some time.

I like this idea, I'll go ahead and do it here too

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
changes have been applied.

**Pull request merged**